web/src/components/task/task-sidebar/finish-button/finish-button.tsx

```diff
@@ -53,17 +49,11 @@ export const FinishButton: FC<TaskSidebarProps> = ({
 
     const handleFinishValidation = isSplitValidation ? onFinishSplitValidation : onFinishValidation;
 
-    let isDisabled = !isAnnotatable || (isValidation && !allValidated);
-    if (jobType !== 'extensive_coverage') {
```

Comment:
Why was the if statement removed?

Reply:
Looks like it's not needed and it's okay to use 1 logic for all cases. `(isValidation && !touchedPagesCount && !editedPagesCount);` part counted the case incorrectly - in case if all pages are validated and not touched anyhow, it should be possible to click "Finish validation" but this check didn't allow to do this. I've tried to figured out when it was introduced (and the reason of this check) and haven't found any reason to keep it here.